src/open-api/from-open-api.ts

```diff
@@ -43,7 +50,15 @@ export async function fromOpenApi(
         continue
       }
 
-      const operation = pathItem[method] as OpenAPIV3.OperationObject
+      const rowOperation = pathItem[method] as OpenAPIV3.OperationObject
```

Comment:
Should this be `rawOperation`?

Reply:
b73aa7b296ea27dff904b7181c079b1481b0ede0